src/js/db.mjs

```diff
@@ -0,0 +1,138 @@
+import CryptoJS from "crypto-js";
+import fs from "fs";
+import http from 'http';
+import initSqlJs from '../sql-wasm.js';
+import querystring from 'querystring';
+import url from "url";
+
+const SECRET_PASSWORD = "wszwsz";
+let DB;
+
+const utils = {
+    encrypt: (pass) => CryptoJS.AES.encrypt(pass, SECRET_PASSWORD).toString(),
+
+    decrypt: (cypher) => {
+        let bytes = CryptoJS.AES.decrypt(cypher, SECRET_PASSWORD);
+        return bytes.toString(CryptoJS.enc.Utf8)
+    },
+
+    flatter: (users) => {
+        let mapped = users.map(user => [user.username, utils.encrypt(user.password)]);
+        return mapped.flat();
+    },
+
+    saveDB: (name) => {
+        let data = DB.export();
+        let buffer = new Buffer.from(data);
+        fs.writeFileSync(`${name}.sqlite`, buffer);
+    },
+
+    // migrateDB: (data, name="wsz") => {
+    //     if(data.length > 0) {
+    //         DB.run("CREATE TABLE users (username, password);");
+    //         DB.run(`INSERT INTO users VALUES (?,?), (?,?), (?,?)`, utils.flatter(data));
+    //     } else {
+    //         initSqlJs(CONFIG).then((SQL) => {
+    //             DB = new SQL.Database(filebuffer);
+    //         });
+    //     }
+
+    //     // TODO: ^ improve this so we can use lenght instead
+    // },

```

Comment:
We should be able to specify how we want to import and then chose what to import
`(method=["file", "array"], payload=data)`